### DISCUSSION TITLE
How to find each line of text dimensions

### DISCUSSION THREAD

**Discussion by kathimohan** [jsvine/pdfplumber#852] [Q&A]
Is there any way to find each line of text's dimensions i.e., left,top,right and bottom . Atleast top and bottom.

**Comment by jsvine** [comment]
Thanks for the nudge on this; it's a feature I've been wanting to add. Now available in v0.9.0, `pdfplumber` has a `Page.extract_text_lines(...)` method. Give it a shot, and let me know if it suits your needs!

**Reply by kathimohan** [reply]
Thank you very much for your help, but for now fixed my requirement by changing the logic, will check based on need and will reply
